lib/takeaway.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'menu'
+
+class Takeaway 
+
+  attr_reader :menu, :basket, :order_arr
+
+  def initialize(menu:, basket:)
+    @menu = menu
+    @basket = basket
+    @order_arr = []
+    @total = 0
+  end
+
+  def print_menu
+    menu.print
+  end
+
+  def print_basket
+    basket.print
+  end
+
+  def add_to_basket(dish, quantity)
+    quantity.times{ @order_arr << dish}
```

Comment:
I believe there is a missing space in the curly brackets here (after dish) but otherwise it's fine. Good method names too :)